From 5c7021db56b652a33b563877cf44419d08cee32c Mon Sep 17 00:00:00 2001
From: Sergey Fedorov <vital.had@gmail.com>
Date: Wed, 9 Oct 2024 18:13:17 +0700
Subject: [PATCH 01/10] mm-netlink.c: fix include order

Some macOS versions require sys/socket.h to be included prior to net/if.h, otherwise it fails to compile.
---
 src/mm-netlink.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mm-netlink.c b/src/mm-netlink.c
index c54942be..58c3d010 100644
--- a/src/mm-netlink.c
+++ b/src/mm-netlink.c
@@ -19,9 +19,9 @@
 
 #include <linux/netlink.h>
 #include <linux/rtnetlink.h>
-#include <net/if.h>
 #include <sys/socket.h>
 #include <sys/types.h>
+#include <net/if.h>
 
 #include <config.h>
 
-- 
2.46.1

